js/siteorigin-panels/view/widget.js

```diff
@@ -278,6 +283,18 @@ module.exports = Backbone.View.extend( {
 
 		// Lets also add the contextual menu for the entire row
 		this.cell.buildContextualMenu( e, menu );
+	},
+
+	/**
+	 * Handler for any action after the user adds a new widget.
+	 * @param widget
+	 */
+	afterUserAddsWidgetHandler: function( widget ) {
+		if( this.model === widget && panelsOptions.instant_open ) {
+			setTimeout(function(){
```

Comment:
Ok, last thing. 😛 Is`setTimeout` really necessary? Any way we could avoid using it and use events and/or callbacks somehow?

Reply:
One thing I did change (and maybe this is what you meant) was using `setTimeout(this.editHandler, 350);`. Much better.

Reply:
Ah ok, all good then.